src/solc_0.5/Land/erc721/ERC721BaseToken.sol

```diff
@@ -406,5 +406,5 @@ contract ERC721BaseToken is ERC721Events, SuperOperators, MetaTransactionReceive
 
     // Empty storage space in contracts for future enhancements
     // ref: https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/issues/13)
-    uint256[49] private __gap;
+    uint256[50] private __gap;
```

Comment:
Updated to 50, it would get decremented when new variables are added: https://ethereum.stackexchange.com/questions/126671/upgradeble-openzeppelin-contracts-why-the-gap-is-needed-if-you-can-add-varia